### PR TITLE
Publish gradle plugins to Sonatype

### PR DESCRIPTION
**Pull request by iNikem**

### CODE REVIEW

**Review by mateuszrzeszutek — APPROVED** [review]

File: .github/workflows/gradle-plugins.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+name: CI build
```
Comment (trask) [inline .github/workflows/gradle-plugins.yml:1]:
```suggestion
name: Publish Gradle plugin snapshots
```

**Review by trask — COMMENTED** [review]
It looks like groupId is `io.opentelemetry.instrumentation.gradle`, should we change that to `io.opentelemetry.instrumentation` based on https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/3555#discussion_r669317787

**Comment by trask** [comment]
or `io.opentelemetry.javaagent`, since it's javaagent-specific..

**Comment by iNikem** [comment]
> or `io.opentelemetry.javaagent`, since it's javaagent-specific..

Done

**Review by mateuszrzeszutek — APPROVED** [review]

**Pull request merged**